src/integrationTest/java/uk/gov/hmcts/darts/cases/controller/CaseControllerGetEventByCaseIdTest.java

```diff
@@ -101,6 +114,8 @@ void casesGetEventsEndpoint() throws Exception {
         expectedJson = expectedJson.replace("<hearing-id>", hearingEntity.getId().toString());
         JSONAssert.assertEquals(expectedJson, actualJson, JSONCompareMode.NON_EXTENSIBLE);
 
+        // assert that we only ever got one event. The one that was associated to the first case hearing
+        Assertions.assertEquals(1, eventEntityList.size());
```

Comment:
We should also verify we get the correct event returned.

Reply:
Donot we already do that on line 113?

Reply:
Yeh, I think this is fine.